docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,99 @@
+- Menus in an application are blank.
+
+You may be able to fix this by clearing KDE / Plasma cache files, by following the steps below.
+
+:::caution
+
+You should not be logged into your Plasma session while doing this. It is also highly recommended to back up the folders mentioned below before deleting anything.
+
+:::
+
+1. While still logged into your Plasma session, cd to your home directory:
+  ```bash
+  cd
+  ```
+1. Run these commands to back up the following directories:
+  ```bash
+  cp -av ~/.kde{,.bak}
+  cp -av ~/.config{,.bak}
+  cp -av ~/.local{,.bak}
+  cp -av ~/share{,.bak}
+  ```
+  The output will show what files were copied. Each directory will have another copy with a .bak extension such as `~/.config.bak`. These can be used to recover configuration files, if necessary.
+1. Shut down the system.
+1. Boot from a live USB, which you can create from one of the [Solus installers](https://getsol.us/download/).
+2. [Mount your hard drive's file system](./boot-rescue.md#mounting-your-system).
+3. Change to your user's directory where you mounted your file system. For example, if you used `/mount`:
+  ```bash
+  cd /mount/home/username
+  ```
+  Replace "username" with the username you use for Solus.
+  You can make sure the directory is correct by listing the files:
+  ```bash
+  ls -laH
+  ```
+  You should see the files you would usually see in your home directory when logged into Solus normally.
+1. Now you need to remove the various caches that Plasma uses.
+  Execute the following commands:
+  ```bash
+  rm -rf .kde
+  rm -rf .cache/plasmashell*
+  rm -rf .cache/org.kde.dirmodel-qml.kcache
+  rm -rf .cache/kioexec/ .cache/krunner/ .cache/ksycoca5*
+  rm -rf .cache/krunnerbookmarkrunnerfirefoxdbfile.sqlite
+
+  cd ../.local/
+  rm -rf kate/ kded5/ klipper/ knewstuff3/ kscreen/ konsole/ kwalletd/ ksysguard/ kmail2/ kcookiejar/ kactivitymanagerd/
+
+  cd ../share/
+  rm -rf dolphin kate kcookiejar kded5 keyrings klipper kmail2 knewstuff3 konsole kscreen ksysguard kwalletd kxmlgui5 plasma_engine_comic plasma plasma_notes org.kde.gwenview
+
+  cd ../.config/
+  rm -f plasma*
+  rm -rf kde* akonadi* KDE kconf_updaterc baloo* dolphinrc drkonqirc gwenviewrc kmail2rc k*rc katemetainfos
+  ```
+1. Shut down the live environment and remove the live USB drive.
+1. Reboot back into your regular Plasma session. Verify the issue is fixed.
+1. It should not be necessary to recover the cache files that were deleted. If you need to, you can recover a file or directory by copying it from the backup directory to the actual directory.
+  For example, to recover the file `.kde/share/config/kdeglobals`:
+  ```
+  cp -av .kde.bak/share/config/kdeglobals .kde/share/config/kdeglobals
+  ```
+  To restore an entire directory:
+  ```
+  cp -rav .kde.bak/share/config/ .kde/share/config
+  ```
+  After restoring the file(s) or directories, log out of Plasma and log back in.
```

Comment:
[Avoid using (s), always](https://developers.google.com/style/plurals-parentheses). I personally prefer always defaulting to plural. Also, saying just "log out" or "restart your system" should be enough (it goes without saying they need to log back in to continue their work)

```suggestion
  After restoring the files or directories, restart your system.
```

Reply:
Fixed